src/components/tasks/IndividualTasks.js

```diff
@@ -5,12 +5,14 @@ const IndividualTaskTableHeaders = () => [
   <FormattedMessage module="individual" id="individual.firstName" />,
   <FormattedMessage module="individual" id="individual.lastName" />,
   <FormattedMessage module="individual" id="individual.dob" />,
+  <FormattedMessage module="individual" id="individual.location" />,
 ];
 
 const IndividualTaskItemFormatters = () => [
   (individual) => individual?.first_name,
   (individual) => individual?.last_name,
   (individual) => individual?.dob,
+  (individual) => individual?.location_id,
```

Comment:
@sniedzielski any thoughts on how to display the location information to be more user-friendly? I'm not sure it's worth adding one or two separate API calls to fetch the location (possibly two if the individual's location switched from one to another)

Reply:
Usually locations in openIMIS are displayed by their code for example if we have 4 level location and:
a) District A
b) Region B
c) Municipality C 
d) Village D 
- location code for Village D is shown as 'ABCD' like in this searcher example 
![Screenshot from 2024-12-17 12-57-37](https://github.com/user-attachments/assets/a1c33ac6-adcf-485d-95b8-db82ca6d0936)
![Screenshot from 2024-12-17 13-31-30](https://github.com/user-attachments/assets/0a2d7143-cb74-4d8d-a440-ea05665849cc)

The other approach in on the insurees page in result list:
![Screenshot from 2024-12-17 12-57-37](https://github.com/user-attachments/assets/13111e7c-ae67-4323-9be0-962bd1544e4a)

Personally I would recommend first approach. In that case in 'formatter' we have to pass location as the 'code' not id. I think somewhere in backend it could be possible to do this in task creation. I have to double check this.

in policyholder graphQL projection we have 'locations{id, uuid, code, name, type, parent{id,uuid,code,name,type,parent{id,uuid,code,name,type,parent{id,uuid,code,name,type}}}}' this kind of definition to fetch location informations. I saw in prev PRs that you used 'location Code' in managing locations. Maybe we should use the same approach, take this code from backend and display here in 'task definition'.